build.sbt

```diff
@@ -1770,14 +1770,23 @@ lazy val `bench-libs` = (project in file("tools/bench-libs"))
     frgaalJavaCompilerSetting,
     assembly / mainClass := (Compile / run / mainClass).value,
     libraryDependencies ++= jmh ++ Seq(
-      "org.openjdk.jmh" % "jmh-core"                  % jmhVersion,
-      "org.openjdk.jmh" % "jmh-generator-annprocess"  % jmhVersion,
-      "org.graalvm.sdk" % "graal-sdk"                 % graalMavenPackagesVersion   % "provided",
+      "org.netbeans.api" % "org-openide-util-lookup"   % netbeansApiVersion,
```

Comment:
Are you using only `@ServiceProvider` annotation from the [lookup library](https://bits.netbeans.org/dev/javadoc/org-openide-util-lookup/overview-summary.html)? Then it is enough to have just a `% "provided"` dependency.

Reply:
Yes, I am using only the `@ServiceProvider` annotation. I have changed the dependency to provided in df993839d0142255b8c90f8658769120c1f550af